docs/manual/java/guide/services/ServiceClients.md

```diff
@@ -4,11 +4,15 @@ We've seen how to define service descriptors and how to implement them, now we n
 
 ## Binding a service client
 
-The first thing necessary to consume a service is to bind it, so that Lagom can provide an implementation for your application to use.  This can be done using the `bindClient` method on [ServiceClientGuiceSupport](api/index.html?com/lightbend/lagom/javadsl/client/ServiceClientGuiceSupport.html).  If you're already binding a service implementation using [ServiceGuiceSupport](api/index.html?com/lightbend/lagom/javadsl/server/ServiceGuiceSupport.html), this interface extends `ServiceClientGuiceSupport`, so your existing module can be used as is:
+The first thing necessary to consume a service is to bind it, so that Lagom can provide an implementation for your application to use.  This can be done using the `bindClient` method on [ServiceClientGuiceSupport](api/index.html?com/lightbend/lagom/javadsl/client/ServiceClientGuiceSupport.html).  
 
 @[bind-hello-client](code/docs/services/client/Module.java)
 
-Note that when you bind a server using `bindServices`, this will automatically bind a client for that service as well.
```

Comment:
This section of the docs discusses binding clients to consume 3rd party services. A reader in this section is probably not indeterested in `bindServices` at the moment.

I think it's an implementation detail not relevant for the end-user which introduces confusion: "I'm  `binding services` why would a client to myself be necessary?"

Hmm, on second thought, that sentence could come in handy on the following code sample... I'll restore it there.